Detectors/TRD/qc/include/TRDQC/Tracking.h

```diff
@@ -83,12 +86,14 @@ struct TrackQC {
   std::array<int, constants::NLAYER> trackletPositionSigned{}; ///< the raw position from Tracklet64 (signed integer)
   std::array<int, constants::NLAYER> trackletDet{};            ///< the chamber of the tracklet
   // some tracklet details to identify its global MCM number to check if it is from noisy MCM
-  std::array<int, constants::NLAYER> trackletHCId{};                                     ///< the half-chamber ID of the tracklet
-  std::array<int, constants::NLAYER> trackletRob{};                                      ///< the ROB number of the tracklet
-  std::array<int, constants::NLAYER> trackletMcm{};                                      ///< the MCM number of the tracklet
-  std::array<float, constants::NLAYER> trackletChi2{};                                   ///< estimated chi2 for the update of the track with the given tracklet
-  std::array<std::array<int, constants::NCHARGES>, constants::NLAYER> trackletCharges{}; ///< charges of tracklets
-  ClassDefNV(TrackQC, 4);
+  std::array<int, constants::NLAYER> trackletHCId{};                                          ///< the half-chamber ID of the tracklet
+  std::array<int, constants::NLAYER> trackletRob{};                                           ///< the ROB number of the tracklet
+  std::array<int, constants::NLAYER> trackletMcm{};                                           ///< the MCM number of the tracklet
+  std::array<float, constants::NLAYER> trackletChi2{};                                        ///< estimated chi2 for the update of the track with the given tracklet
+  std::array<std::array<int, constants::NCHARGES>, constants::NLAYER> trackletCharges{};      ///< charges of tracklets
```

Comment:
Has the whole rule of not doing arrays of arrays been dropped?

Reply:
I thought the problem were variable-size containers, e.g. vectors. Arrays as fixed size containers should be ok, and I think our digits actually contain an array.

Reply:
I dont remember hence my query.